setup.ts

```diff
@@ -247,15 +247,11 @@ async function recaptcha(): Promise<void> {
   }
 }
 async function recaptchaSiteKey(): Promise<void> {
-  console.log(
-    "\nPlease visit this URL to set up reCAPTCHA:\n\nhttps://www.google.com/recaptcha/admin/create"
-  );
-  console.log(
-    '\nSelect reCAPTCHA v2 and the "I`m not a robot" checkbox option'
-  );
-  console.log(
-    '\nAdd "localhost" in domains and accept the terms, then press submit'
-  );
+  if (process.env.RECAPTCHA_SECRET_KEY) {
```

Comment:
If you enter the site key and it's accepted the value gets immediately updated in the `.env` file. If you make a mistake and hit <CTRL-C> then your mistake is recorded in the config. There must be a way to confirm all the values before committing them to the configuration file

![image](https://github.com/PalisadoesFoundation/talawa-api/assets/16875803/2615c49f-1946-4a24-824d-fb7267806a94)